src/__test__/index.test.ts

```diff
@@ -33,4 +35,18 @@ describe('exports', () => {
 
 		expect(swup.version).not.toBeUndefined();
 	});
+
+	it('passes relative URL to ignoreVisit', () => {
+		let ignorableUrl = 'nothing';
+		const swup = new Swup({
+			ignoreVisit: (url, { el } = {}) => {
+				ignorableUrl = url;
+				return false;
+			}
+		});
+
+		swup.shouldIgnoreVisit(baseUrl + '/path/?query#hash');
```

Comment:
Just an idea. Feels like calling `swup.loadPage` is what really matters when it comes to ignoring navigation. Could we add a test that `shouldIgnoreVisit` is even being called when the `loadPage` is hit, and that it prevents it? e2e test is there too tho, so not important and feel free to skip.